_jekyll/_layouts/default.html

```diff
@@ -12,7 +12,13 @@
         {% endif %}
     </head>
 
-    <body style="margin: 1rem;">
-{{ content }}
+    <body>
+      {% if page.title == 'Footer' %}
+      {{ content }}
+      {% else %}
+      <div style="margin: 1rem;">

```

Comment:
This mark up will be displayed in the foot example code snippet. Also by removing the margin from all over patterns the edges of the patterns will be hidden.